src/_listBase.scss

```diff
@@ -100,6 +100,19 @@
       }
     }
 
+    &--inactive {
+      pointer-events: none;
+
+      .#{$block}__title,
+      .#{$block}__secondary {
+        pointer-events: none;
+      }
+    }
+
+    &--no-seprator {
```

Comment:
couldn't find this anywhere in the code as an example

Reply:
added examples for all css changes.